packages/inventory/src/Stocktake/ListView/Toolbar.tsx

```diff
@@ -5,19 +5,33 @@ import {
   useTranslation,
   DeleteIcon,
   AppBarContentPortal,
-  SearchBar,
   FilterController,
+  StocktakeNodeStatus,
```

Comment:
offroad! 😁 

The filter for stocktakes has changed, and no longer supports filtering by comment..
I've popped in a quick filter by status instead (nothing else available looked useful) - though the API only supports `equalTo` so I couldn't implement a multiple option version. not a problem here, since there's only two status values 😆